ownCloud/Viewer/PDFThumbnailCollectionViewCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  PDFThumbnailCollectionViewCell.swift
+//  ownCloud
+//
+//  Created by Michael Neuwert on 04.10.2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import PDFKit
+
+class PDFThumbnailCollectionViewCell: UICollectionViewCell {
+    static let identifier = "PDFThumbnailCollectionViewCell"
+
+    fileprivate let pageLabelBottomMargin: CGFloat = 5
+    fileprivate let pageLabelHeightMultiplier: CGFloat = 0.1
+
+    var imageView: UIImageView?
+    var pageLabel: UILabel?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupSubviews()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupSubviews()
+    }
+
+    fileprivate func setupSubviews() {
+        self.backgroundColor = UIColor.lightGray
+
+        imageView = UIImageView()
+        imageView?.translatesAutoresizingMaskIntoConstraints = false
+        imageView?.contentMode = .scaleAspectFit
+        self.contentView.addSubview(imageView!)
+
+        pageLabel = UILabel()
+        pageLabel?.backgroundColor = UIColor.clear
```

Comment:
Page labels are difficult to read if the page is darker. As far as I can see, Apple Books uses a white (or bright gray?) color here so the page number remains legible even on dark backdrops. Getting that "label" look of Apple Books, though, may require another container view around the label.